src/tests/mocks/upgrades_mocks.cairo

```diff
@@ -12,12 +12,6 @@ trait IUpgradesV1<TState> {
     fn remove_selector(self: @TState);
 }
 
-trait UpgradesV1Trait<TState> {
-    fn set_value(ref self: TState, val: felt252);
-    fn get_value(self: @TState) -> felt252;
-    fn remove_selector(self: @TState);
-}
-
```

Comment:
why are we removing the traits?

Reply:
We were only using the upgrades mock traits to define the v1 and v2 impls, but it seems cleaner to just define impls of the v1 and v2 interfaces